src/App.vue

```diff
@@ -34,6 +34,12 @@ export default {
             if (themePref == "auto") this.theme = darkModePreference.matches ? "dark" : "light";
             else this.theme = themePref;
 
+            // Change title bar color to white
+            if (this.theme === "light") {
+                const themeColor = document.querySelector("meta[name='theme-color']");
+                themeColor.setAttribute("content", "#FFF");
+            }
```

Comment:
Don't we need to set it back to dark if the theme changes back to dark? (eg: when using auto mode)

Reply:
The `theme-color` meta tag by default uses the dark theme, therefore if dark theme is set the condition would be false and the theme color won't be set which means a fallback to the default value.

Reply:
Another approach would be injecting the meta tag conditionally based on user's theme.

Reply:
What if this scenario:

- The user has auto theme mode and is currently using dark mode
- The user changes their theme from their system preferences to light, and then to dark (without reloading the page)

Won't it remain light in such a scenario?

Reply:
You're absolutely right, forgot about this scenario.
Next commit will fix this.